benchmarks/user/benchmark.py

```diff
@@ -66,7 +66,7 @@ def default_list(ctx, param, value):
                      help='Number of grid points along each axis'),
         click.option('-s', '--spacing', default=(20., 20., 20.),
                      help='Spacing between grid sizes in meters'),
-        click.option('-n', '--nbpml', default=10,
+        click.option('-n', '--nbl', default=10,
```

Comment:
change help too to what 'nbl' means

Reply:
waiting for reply from [here](https://github.com/opesci/devito/issues/972#issue-510485866)

Reply:
Number of absorbing layers

Reply:
or

Number of absorbing ("sponge") layers